### PR TITLE
docs: getting started zkApps

### PR DESCRIPTION
**Pull request by barriebyron**
For #630 
This PR 
- creates a standalone [zkApps Getting Started](https://docs2-git-zkapp-getting-started-minadocs.vercel.app/zkapps/getting-started-zkapps) page: a one-page zkApp developer landing page that MF will share everywhere to help our zkApp devs get a solid start, thanks @ymekuria for the core developer workflow in https://github.com/o1-labs/zkapp-cli/issues/469 
- moves the ultra high-level content from [zkApps Overview > Quickstart](https://docs.minaprotocol.com/zkapps#quickstart) to this page to separate explainer concepts and dev-focused steps (and prevents duplication)
- moves the installation content from https://docs.minaprotocol.com/zkapps#quickstart from overview to a new installation page to create [zkApp CLI Installation](https://docs2-git-zkapp-getting-started-minadocs.vercel.app/zkapps/install-zkapp-cli) a canonical installation and upgrade page for zkApp CLI (long overdue, imo)

audience: devs who want to use zkApps and zkApp CLI

### CODE REVIEW

**Review by nicc — CHANGES_REQUESTED** [review]

File: docs/zkapps/getting-started-zkapps.mdx [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+### 4. Add testing code
+
+When you use the zkApp CLI to create a project, tests and examples are included.
+
+1. Import the smart contract and `o1js` into your test file. 
+
+    See the `import` statements in the [Add.test.ts](https://github.com/o1-labs/zkapp-cli/blob/main/templates/project-ts/src/Add.test.ts#L1-L2) example file.
+
+1. Create a local blockchain instance to interact with.
```
[the remark this answers is not in the document]
Comment (barriebyron) [reply docs/zkapps/getting-started-zkapps.mdx:83]: @nicc  happy to make changes when I know something is a code literal - sometimes, I just don't know (yet)

File: docs/zkapps/getting-started-zkapps.mdx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+    See the local blockchain code in the [Add.test.ts](https://github.com/o1-labs/zkapp-cli/blob/main/templates/project-ts/src/Add.test.ts#L27-L28) example file.
+
+1. Initialize test accounts to pay fees and sign transactions. o1js provides an array of 10 accounts you can start using right away.
+
+    See the initialize account code in the [Add.test.ts](https://github.com/o1-labs/zkapp-cli/blob/main/templates/projec%C3%A5%C3%A7t-ts/src/Add.test.ts#L29-L31) example file.
+
+1. Deploy the smart contract to the local blockchain.
+
+    See [Deploying a contract locally](https://docs.minaprotocol.com/zkapps/how-to-test-a-zkapp#deploying-a-contract-locally).
```
[the remark this answers is not in the document]
Comment (barriebyron) [reply docs/zkapps/getting-started-zkapps.mdx:93]: TIL copy-pasta

**Review by ymekuria — APPROVED** [review]
This is a work in progress and we will continue to iterate to make this more comprehensive.

**Comment by barriebyron** [comment]
We look forward to getting feedback from devs! This Getting Started deliverable is the first iteration of awesomeness to grow and improve!

File: docs/zkapps/getting-started-zkapps.mdx [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+### 3. Add testing code
+
+When you use the zkApp CLI to create a project, tests and examples are included. 
+
+1. See the `import` statements in the [Add.test.ts](https://github.com/o1-labs/zkapp-cli/blob/main/templates/project-ts/src/Add.test.ts#L1-L2) example file.
+
+1. A `LocalBlockchain` instance to interact with is included in the [Add.test.ts](https://github.com/o1-labs/zkapp-cli/blob/main/templates/project-ts/src/Add.test.ts#L27-L28) example file.
+
+1. In o1js, an array of 10 test accounts to pay fees and sign transactions are provided for the `LocalBlockchain` instance in the `Local.testAccounts;` in the [Add.test.ts](https://github.com/o1-labs/zkapp-cli/blob/main/templates/project-ts/src/Add.test.ts#L29-L31) example file. The example accesses two of these public/private key pair accounts. The example uses these names, but the names can be anything:
```
Comment (nicc) [inline docs/zkapps/getting-started-zkapps.mdx:71]:
```suggestion
1. In o1js, an array of 10 test accounts to pay fees and sign transactions are provided for the `LocalBlockchain` instance. These can be accessed with `Local.testAccounts` as per the [Add.test.ts](https://github.com/o1-labs/zkapp-cli/blob/main/templates/project-ts/src/Add.test.ts#L29-L31) example file. The example uses the public/private key pairs of two of these accounts. They are given these names, but the names can be anything:
```

File: docs/zkapps/getting-started-zkapps.mdx [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+1. Deploy the smart contract to the `LocalBlockchain` instance that simulates a network for testing purposes.
+
+    See the `localDeploy` function in the [Add.test.ts](https://github.com/o1-labs/zkapp-cli/blob/main/templates/project-ts/src/Add.test.ts#L38-L46) example file.
+
+### 4. Add functionality
+
+Add the logic for your smart contract.
+
+1. Start experimenting with iterative development to build and test one method at a time. Add functionality to the smart contract by implementing an `@method`.
```
Comment (nicc) [inline docs/zkapps/getting-started-zkapps.mdx:84]:
```suggestion
1. Start experimenting with iterative development to build and test one method at a time. Add functionality to the smart contract by implementing a `@method`.
```
I think? Feels better to me anyway

File: docs/zkapps/getting-started-zkapps.mdx [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+
+- Create a Node.js script to run the smart contract and test it's functionality, similar to step [3. Add testing code](#3-add-testing-code).
+
+    For an example, see the Node.js script that runs the Tic Tac Toe smart contract in the [run.ts](https://github.com/o1-labs/zkapp-cli/blob/main/examples/tictactoe/ts/src/run.ts) file.
+
+### 6. Test with a live network
+
+To deploy the smart contract to the Testnet, run the `zk` commands from the directory that contains your smart contract. 
+
+1. Configure your zkapp.
```
Comment (nicc) [inline docs/zkapps/getting-started-zkapps.mdx:111]:
```suggestion
1. Configure your zkApp.
```

**Review by nicc — APPROVED** [review]

**Pull request merged**